src/ontology/edits/oeo-physical.omn

```diff
@@ -3402,6 +3405,20 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/755",
         <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000117
     
     
+Class: OEO_00010093
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A river is liquid water that moves through permanent or seasonal flow process from elevated land towards lower elevations through a definite channel and empties either into a sea, lake, or another river or ends on land as bed seepage and evapotranspiration exceed water supply.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/760
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/765",
+        rdfs:label "river"@en,
+        owl:equivalentClass "ENVO:00000022"
```

Comment:
The OWL vocabulary lists `rdfs:Class` as the range of `owl:equivalentClass`, so we should use the IRI instead of a literal here. (the IRI is http://purl.obolibrary.org/obo/ENVO_00000022 in this case)

Reply:
Thanks